tests/common/aws_test_runner.c

```diff
@@ -188,6 +188,7 @@ static void RunTests( void )
     #if ( testrunnerFULL_SERIALIZER_ENABLED == 1 )
         RUN_TEST_GROUP( Full_Serializer_CBOR );
         RUN_TEST_GROUP( Full_Serializer_JSON );
+        RUN_TEST_GROUP( Deserializer_JSON );
```

Comment:
Our test infra needs to be updated correspondingly. I am not sure where "Serializer Tests" are defined..

Reply:
Test rules are defined here : https://github.com/amazon-freertos/afr-host-tools-config/blob/40e545ace0c2885330235b49c4a806415359d002/test_rules.yml

I think its better to prefix with Full_Serializer so that it runs with all other tests.

Reply:
@ravibhagavandas, I updated the PR to put the new tests behind the Full_Serializer prefix.